src/Sylius/Behat/Context/Api/Shop/AccountContext.php

```diff
@@ -15,92 +15,44 @@
 
 use Behat\Behat\Context\Context;
 use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\Request;
 use Sylius\Behat\Client\ResponseCheckerInterface;
-use Sylius\Behat\Context\Setup\ShopSecurityContext;
 use Sylius\Behat\Service\SharedStorageInterface;
-use Sylius\Component\Core\Model\OrderInterface;
-use Sylius\Component\Core\Model\ShopUserInterface;
-use Symfony\Component\HttpFoundation\Response;
 use Webmozart\Assert\Assert;
 
+
 final class AccountContext implements Context
```

Comment:
Why did you separate this context and the next one?, IMO Both are related with customer account

Reply:
One is related with Customer things and the 'password change' is related to shopUser ( who is not customer)